src/stacks-hierarchy/Root_TicketAssistantStack/TicketAssistant_CategoryPickerScreen.tsx

```diff
@@ -65,7 +65,7 @@ export const TicketAssistant_CategoryPickerScreen = ({
 
   useEffect(() => {
     return unsubscribe;
-  }, [navigation]);
+  }, [unsubscribe]);
```

Comment:
I am not sure what this actually does the useEffect 😓, look for something to change?